profile/src/main/kotlin/com/imashnake/animite/profile/ProfileScreen.kt

```diff
@@ -66,17 +76,25 @@ fun ProfileScreen(
     expiresIn: Int? = null
 ) {
     accessToken?.let { viewModel.setAccessToken(it) }
+    viewModel.refreshViewer(false)
```

Comment:
Is this needed if the saved state handle flow in the VM is false by default?

Reply:
I think it's needed so that `savedStateHandle[REFRESH]` is set back to false after a refresh. If it stays true, consecutive refreshes don't work.

Reply:
So Jack's idea of a MutableSharedFlow<Unit> (or Channel) is less likely to run into these issues, and wouldn't need this extra line because there's no equality check? 👀

Reply:
Fixed in #133.